### PR TITLE
Bug 1032681 - copy selected file if package.json has gaia_shared

### PR DESCRIPTION
**Pull request by cctuan**
https://bugzilla.mozilla.org/show_bug.cgi?id=1032681

### CODE REVIEW

**Comment by try-server-hook** [comment]
[cctuan George (cctuan)](https://github.com/cctuan) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=89c6eb7cd5e1)

**Comment by try-server-hook** [comment]
[cctuan George (cctuan)](https://github.com/cctuan) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=ada223786eaf)

**Comment by try-server-hook** [comment]
[cctuan George (cctuan)](https://github.com/cctuan) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=210bde0ac544)

File: build/webapp-shared.js [hunk 1]
```diff
@@ -388,15 +388,18 @@ WebappShared.prototype.copyShared = function() {
...
   var self = this;
-  var sharedDataFile = this.webapp.buildDirectoryFile.clone();
-  sharedDataFile.append('gaia_shared.json');
-  if (sharedDataFile.exists()) {
-    var sharedData = JSON.parse(utils.getFileContent(sharedDataFile));
-    Object.keys(sharedData).forEach(function(kind) {
-      sharedData[kind].forEach(function(path) {
-        self.pushFileByType(kind, path);
+  var packageJson = this.webapp.appDir.clone();
+  packageJson.append('package.json');
```
Comment (RickyChien) [inline build/webapp-shared.js]: We should get rid of the pattern ```append()```. Use 
```
var packageJson = utils.getFile(this.webapp.appDir.path, 'package.json');
```

**Comment by try-server-hook** [comment]
[cctuan George (cctuan)](https://github.com/cctuan) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=9adb04d0158f)

**Pull request closed without merging**